### ISSUE TITLE
Accessibility : folds should be identified by sound, similar to how breakpoints identified

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#13952]
One of the use cases for me when considering vsCode is to have a folding editor. This makes reading someone else's code far easier when using a screen reader.

However, it is currently not possible to identify a fold when moving through the file. I can initiate folding and when viewing my own code, I obviously know that a function that looks empty is "folded".

SUggest playing a cound when entering (and possibly when exiting) a folded region when accessibility mode is enabled. I noticed that breakpoints play a sound when the line containing them is reached; similar thing could be done for folds.

Thanx.
-- Rich
